client/scss/components/_modals.scss

```diff
@@ -148,4 +148,4 @@ $zindex-modal-background: 500;
     max-width: 100em;
     padding: 0 0 2em;
   }
-}
+}
```

Comment:
please ensure your editor is set up correctly, it looks like you have removed the End of file line break here.